dev/com.ibm.ws.cdi.beansxml_fat/fat/src/com/ibm/ws/cdi12/suite/FATSuite.java

```diff
@@ -48,7 +48,7 @@
 public class FATSuite {
 
     @ClassRule
-    public static RepeatTests r = RepeatTests.withoutModification().andWith(FeatureReplacementAction.EE8_FEATURES());
+    public static RepeatTests r = RepeatTests.withoutModification().andWith(FeatureReplacementAction.EE8_FEATURES()).andWith(FeatureReplacementAction.EE9_FEATURES());
```

Comment:
Is it correct to run against all versions in LITE mode?

Reply:
No ... plan to do a follow up PR to continue tidying/refactoring